packages/go_router_builder/lib/src/type_helpers.dart

```diff
@@ -149,8 +203,13 @@ class _TypeHelperBigInt extends _TypeHelperWithHelper {
   String helperName(DartType paramType) => 'BigInt.parse';
 
   @override
-  String _encode(String fieldName, DartType type) =>
-      '$fieldName${type.ensureNotNull}.toString()';
+  String _encode(String fieldName, DartType type, String? customEncoder) {
+    String encode = '$fieldName${type.ensureNotNull}.toString()';
```

Comment:
nit: this could be:  
```
   if (customEncoder != null) {
      return  '$customEncoder($encode)';
    }    
return  '$fieldName${type.ensureNotNull}.toString()';
```